src/Lucid/Relations/Parser.js

```diff
@@ -21,17 +21,32 @@ const CE = require('../../Exceptions')
  */
 class RelationParser {
   /**
+   * Returns the relationships object
+   *
+   * @method _normalizeRelations
+   *
+   * @param {Object|Array}             relations
```

Comment:
May you change `Array` to `Object[]`

Reply:
May be `Array` to `String[]` not `Object[]`?

Reply:
I didn't read the code, so if you receive an array of string here yes, `string[]` will be better 😄